src/cityreader/cityreader.py

```diff
@@ -5,24 +5,48 @@
 # We have a collection of US cities with population over 750,000 stored in the
 # file "cities.csv". (CSV stands for "comma-separated values".)
 #
-# In the body of the `cityreader` function, use Python's built-in "csv" module 
+# In the body of the `cityreader` function, use Python's built-in "csv" module
 # to read this file so that each record is imported into a City instance. Then
 # return the list with all the City instances from the function.
-# Google "python 3 csv" for references and use your Google-fu for other examples.
+# Google "python3 csv" for references and use your Google-fu for more examples.
 #
 # Store the instances in the "cities" list, below.
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
+import sys
+
+
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = str(name)
+        self.lat = float(lat)
+        self.lon = float(lon)
+
+    def __repr__(self):
+        return f"<City: {self.name}, Lat: {self.lat}, Lon: {self.lon}>"
+
+
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    cities = cities
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+    with open('cities.csv') as csvfile:
+        read_cities = csv.DictReader(csvfile)
+        for row in read_cities:
+            try:
+                cities.append(City(row['city'], row['lat'], row['lng']))
+            except:
+                pass
+
```

Comment:
Nice use of csv.DictReader.  In the use of csv.DictReader, the try/except wouldn't be needed for the purpose of getting past the first row of titles (but still creates some form of safety net that is not triggered in this circumstance).

If you were using csv.reader instead, the try/except could be used to handle the error from the first line of titles (since you would need to try to cast lat/lon rows as float() in that circumstance, it will error on the title line).  Or similarly could use the next() function on the iterator before the for loop to pass that first row.

Great Work!